### PR TITLE
Add precious material selection to Counterclockwork Focus

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: packs/feats/counterclockwork-focus.json [hunk 1]
```diff
@@ -35,35 +35,65 @@
...
-                "option": "counterclockwork",
+                "option": "counterclockwork-focus",
                 "toggleable": true
             },
             {
+                "disabledIf": [
+                    {
+                        "not": "counterclockwork-focus"
+                    }
+                ],
```
Comment (TikaelSol) [inline packs/feats/counterclockwork-focus.json:47]: This should be able to be done as one RollOption RE. A suboption will also set the base option when toggled on.

Comment (pedrogrullada) [reply packs/feats/counterclockwork-focus.json:47]: Yeah, but treating the ammunition as a precious material is optional. I suppose I can't think of a reason you wouldn't do so right now, but that was the rationale. I can make it one toggle.

Comment (TikaelSol) [reply packs/feats/counterclockwork-focus.json]: Make a `none` option?

Comment (pedrogrullada) [reply packs/feats/counterclockwork-focus.json]: Not sure why it didn't occur to me. Done!

File: packs/feats/counterclockwork-focus.json [hunk 2]
```diff
@@ -35,35 +35,60 @@
...
-                        },
-                        {
-                            "start": 18,
-                            "value": {
-                                "diceNumber": 3
-                            }
-                        }
-                    ]
-                }
+                "selector": "strike-damage"
```
Comment (simonward) [inline packs/feats/counterclockwork-focus.json:76]:
```suggestion
                "selector": "firearm-weapon-group-damage",
```

**Pull request merged**